src/IssueViz.Security/Taint/TaintIssuesSynchronizer.cs

```diff
@@ -92,8 +95,10 @@ public async Task SynchronizeWithServer()
                 }
 
                 var projectKey = bindingConfiguration.Project.ProjectKey;
+                var serverBranch = await serverBranchProvider.GetServerBranchNameAsync();
+
                 var taintVulnerabilities = await sonarQubeService.GetTaintVulnerabilitiesAsync(projectKey,
-                    null, // TODO - pass the current branch
+                    serverBranch,
```

Comment:
The actual product changes at this level are very simple.
Hopefully suppressions will be as straightforward...